File: test/data/phenology/crop2ml/algo/cpp/LeafNumberModule.cpp

if (phase >= 1 && phase < 4)
{
	if (hasFlagLeafLiguleAppeared == 0) //sowingToAnthesis
	{
		if (phyllochron == 0.0)
		{
			phyllochron = 0.0000001;
		}
		leafNumber = leafNumber + fmin(deltaTT / phyllochron, 0.999);
	}
}

File: test/data/phenology/crop2ml/algo/f90/LeafNumberModule.f90

    !use crop2mlModules
        IF ((phase .GE. 1.0) .AND. (phase .LT. 4.0)) THEN
            IF (hasFlagLeafLiguleAppeared .EQ. 0) THEN
                IF (phyllochron .EQ. 0.0) THEN
                    phyllochron = 0.0000001
                END IF
                leafNumber = leafNumber + MIN(deltaTT / phyllochron, 0.999)
            END IF
        END IF